.github/workflows/ci-linux.yaml

```diff
@@ -11,16 +11,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-latest']
+        os: ['ubuntu-latest','macos-latest']
         perl: 
           - '5.16'
-          - '5.18'
-          - '5.20'
-          - '5.22'
-          - '5.24'
-          - '5.26'
-          - '5.28'
           - '5.30'
+          - 'latest'
```

Comment:
As long as openSUSE Leap 15 and SLE15 are still supported running 5.26 I suggest to keep at least that version tested but I guess we could get rid of 5.16

Reply:
I think as long as this module is supposed to run on older versions as well, they should be checked in CI.
It's a CPAN module and not just a module for current openSUSE.
Otherwise the minimum perl version should be specified in the dependencies.